tests/rbd/rbd_migration_2diff_ec_pool_ec.py

```diff
@@ -7,6 +7,7 @@
 Test cases covered -
 1) CEPH-83573322 - Migrate the images from 2 different source pools to destination
 and check the behaviour
+2) CEPH-83573323 - Data integrity and Image info wrt associated pool after image migration
```

Comment:
@rkhambad are you including this as part of any of the test suites? 
else it will never run in automation pipeline right?

Reply:
will mark this test case as duplicate as we have included this scenario in same module.
This is for inclusion of data integrity check as the steps are very much similar.